src/components/movement-sensor/client.test.ts

```diff
@@ -8,11 +8,33 @@ import { MovementSensorClient } from './client';
 
 let sensor: MovementSensorClient;
 
+interface Val {
+  toJavaScript(): string;
+}
+
+const mapVals: Val = {
```

Comment:
I'm realizing now that there probably doesn't have to be an interface here, and you could just create `mapVals` directly without a type.

Reply:
good point, I'll remove it